docs/root/configuration/http/http_filters/golang_filter.rst

```diff
@@ -45,6 +45,19 @@ For example, to build the ``.so`` for a ``foo`` plugin, you might run:
 
    $ bazel run @go_sdk//:bin/go build -- --buildmode=c-shared  -v -o path/to/output/libfoo.so path/to/src/foo
 
+API compatibility
```

Comment:
Would it be better to move this to the 'Attention' section of `Developing a Go plugin`.